charmcraft/commands/store/client.py

```diff
@@ -44,11 +45,42 @@
 STORAGE_BASE_URL = 'https://storage.staging.snapcraftcontent.com'
 
 
+def _is_ci_env() -> bool:
+    """Utilize standard env variables to determine if the environment is for testing."""
+    env_prefixes = ["TRAVIS", "AUTOPKGTEST_TMP"]
+    for prefix in env_prefixes:
+        for key in os.environ.keys():
+            if key.startswith(prefix):
+                return True
+    return False
```

Comment:
No need for this function.

The `env_prefixes` can be a global constant (`TESTING_ENV_PREFIXES`?), and the rest of the code can be written like:

     if any(key.startswith(TESTING_ENV_PREFIXES) for key in os.environ.keys()):